### PR TITLE
Add meow/products index and show

### PR DESCRIPTION
**Pull request by ashrafmansur**
This PR adds the products index and show actions specifically for a registered user. This is done under the namespace, meow. I have added meow folders to both the controllers folder as well as the views folder and grouped the relevant files inside. I have also added Faker data in the seeds.rb file.

Would greatly appreciate feedback on my implementation of both the namespace concept as well as the faker gem as it's my first time using both. I hope I have made the code as clear as possible but any suggestions for improvement are welcome as always! Thanks!

### CODE REVIEW

**Review by hadi91 — APPROVED** [review]
Nice work ash. A small suggestion to the logout link. Maybe we can add another confirmation to ask the user if they really wish to logout? Similar to the delete method. Lemme know what you guys think. Cheers

File: app/views/meow/products/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<h1>All Cats</h1>
+
+<% @products.each do |product| %>
+
+<%= product.breed %>
+<%= product.dob %>
```
Comment (ssoonsl) [inline app/views/meow/products/index.html.erb:6]: Looks good! Although I just remembered that we need to compute the litter's age here. We'll do that together later.

File: app/views/meow/products/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+<%= product.description %>
+<%= product.price %>
```
Comment (ssoonsl) [inline app/views/meow/products/index.html.erb:8]: Oh yeah we need to either add the $ or SGD here in the view. And format the price to 2 decimal places.

File: db/seeds.rb [hunk 3]
```diff
@@ -5,3 +5,13 @@
...
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+5.times do
+  Product.create([{
+    breed:        Faker::Cat.breed,
+    dob:          Faker::Date.backward(365),
+    description:  Faker::Lorem.paragraph,
+    price:        Faker::Commerce.price,
+    quantity:     Faker::Number.between(1,5)
```
Comment (ssoonsl) [inline db/seeds.rb]: Great job on not only figuring out Faker, but also managing to get it to fake dates and prices!

**Review by ssoonsl — COMMENTED** [review]
Nice work Ash! We can add the log out confirmation like Hadi suggested. By the way, CircleCI is failing cause we still have the default generated specs for ProductsController even though we moved it to Admin::ProductsController now.

**Pull request merged**